.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import { withFluentProvider, withStrictMode } from '@fluentui/react-storybook';
+import DocsPage from 'storybook-package-context-loader/dist/package-context-docs-page';
```

Comment:
chore: we would like to avoid to use non standard APIs from 3rd party (nested paths) - I'd recommend to decouple your package or provide proper exports mapping.